packages/govern-contract-utils/contracts/protocol/IArbitrable.sol

```diff
@@ -2,14 +2,17 @@
  * SPDX-License-Identifier:    MIT
```

Comment:
Based on our conversation earlier, I'm not sure if these make sense belonging in the utils. I imagine these are not being used outside of `govern-core`?

Reply:
As we talked, I think it makes sense to keep them in core unless we use some utils in several packages

Reply:
Done in b766b64.